src/tests/ftest/util/ior_test_base.py

```diff
@@ -125,8 +127,12 @@ def verify_pool_size(self, original_pool_info, processes):
             original_pool_info (PoolInfo): Pool info prior to IOR
             processes (int): number of processes
         """
+        if isinstance(self.pool, TestPool):
+            self.pool = self.pool.pool
+
         # Get the current pool size for comparison
-        current_pool_info = self.pool.pool.pool_query()
+        # pylint: disable=no-member
+        current_pool_info = self.pool.pool_query()
```

Comment:
It is not a good idea to redefine a self attribute as a different object type.   Assuming the issue here is you want to support self.pool being assigned as either a DaosPool or Test Pool object, then I'd recommend the following:
```
if isinstance(self.pool, TestPool):
    self.pool.get_info()
    current_pool_info = self.pool.info
elif isinstance(self.pool, DaosPool):
    current_pool_info = self.pool.pool_query()
else:
    raise Exception("Unsupported pool object class: {}".format(type(self.pool)))
```

Alternatively you could just pick one object type and require that self.pool is an object of that type.

Reply:
Will do.